src/Toolkit/A.php

```diff
@@ -375,6 +375,19 @@ public static function fill(array $array, int $limit, $fill = 'placeholder'): ar
         return $array;
     }
 
+    /**
+     * A simple wrapper around array_map
+     * with a sane argument order
+     *
+     * @param array $array
+     * @param \Closure $map
+     * @return array
+     */
+    public static function map(array $array, Closure $map): array
```

Comment:
```suggestion
     * @param callable $map
     * @return array
     */
    public static function map(array $array, callable $map): array
```

Should be `callable` to work correctly. For ex: `A::map($array, 'trim');`

Reply:
✅